tests/fixtures/yaml/symfony3.yaml

```diff
@@ -29,3 +29,13 @@ form:
             options:
                 true: true
                 false: false
+        testClass:
+            My\Class\Path: value
+        testQuoting:
+            complex1: 'this is "very" complex quoting'
+            complex2: "this is also 'very' complex quoting"
+            middle: this string is "quoted" in the middle
```

Comment:
How about the above but also just a single quote? Complex2 would pass without being modified because it's a quoted value in it's entirety, and middle would pass because the value would be wrapped in single quotes, but I'm wondering about `middeSingle: this string has some single quote's, and maybe a 'air quote', anything to make your life diff'cult`